docs/operations/cpu_hotplug.md

```diff
@@ -0,0 +1,180 @@
+# CPU Hotplug
+
+The CPU hotplug feature was introduced in KubeVirt v1.0, making it possible to configure the VM workload
+to allow for adding or removing virtual CPUs while the VM is running.
+
+### Abstract
+A **virtual CPU** (vCPU) is the CPU that is seen to the Guest VM OS. A VM owner can manage the amount of vCPUs from the VM spec template using the CPU topology fields (`spec.template.spec.domain.cpu`). The `cpu` object has the integers `cores,sockets,threads` so that the virtual CPU is calculated by the following formula: `cores * sockets * threads`. 
+
+Before CPU hotplug was introduced, the VM owner could change these integers in the VM template while the VM is running, and they were staged until the next boot cycle. With CPU hotplug, it is possible to patch the `sockets` integer in the VM template and the change will take effect right away. 
+
+Per each new socket that is hot-plugged, the amount of new vCPUs that would be seen by the guest is `cores * threads`, since the overall calculation of vCPUs is `cores * sockets * threads`. 
+
+## Configuration
+
+### Enable feature-gate
+In order to enable CPU hotplug we need to add the `VMLiveUpdateFeatures` feature gate in Kubevirt CR:
+
+```yaml
+apiVersion: kubevirt.io/v1
+kind: KubeVirt
+spec:
+  configuration:
+    developerConfiguration:
+      featureGates:
+        - VMLiveUpdateFeatures
+```
+
+### Configure the workload update strategy
+Current implementation of the hotplug process requires the VM to live-migrate.
+The migration will be triggered automatically by the workload updater. The workload update strategy in the KubeVirt CR must be configured with `LiveMigrate`, as follows:
+
+```yaml
+apiVersion: kubevirt.io/v1
+kind: KubeVirt
+spec:
+  workloadUpdateStrategy:
+    workloadUpdateMethods:
+    - LiveMigrate
+```
+
+### Enable in VM spec
+The VM object should look like this:
+```yaml
+apiVersion: kubevirt.io/v1
+kind: VirtualMachine
+spec:
+  liveUpdateFeatures:
+    cpu: {}
+```
+Specifying the `cpu.maxSockets` value is optional. If you leave it unset, it will default to 4 times the `sockets` value.
+
+**NOTE:** In order for these changes to take effect the VM needs to be rebooted.
```

Comment:
Out of this spefic PR but at some point in the future, we should decide whether to write notes as
`**NOTE:**`
or
`*NOTE:*`
or
`NOTE:`
or
`> **Note**`
or
`> *Note:*`
or
`> Note:`

@aburdenthehand FYI